great_expectations/dataset/pandas_dataset.py

```diff
@@ -383,6 +383,13 @@ def expect_column_values_to_not_be_null(self, column,
         # Pass element_count instead of nonnull_count, because that's the right denominator for this expectation
         success, percent_success = self._calc_map_expectation_success(success_count, element_count, mostly)
 
+        return_obj = self._format_column_map_output(result_format, success, element_count, success_count,
+                                                    unexpected_list, unexpected_index_list)
+
+        return_obj['result'].pop('unexpected_percent_nonmissing', None)
```

Comment:
Minor stylistic preference to using `del return_obj['result']['unexpected_percent_nonmissing']` and same for `partial_unexpected_counts` since we're not using the returned values.